### PR TITLE
Update exposures-without-assignments.md

### PR DESCRIPTION
**Pull request by akhil-prakash**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/experiment/guides/troubleshooting/exposures-without-assignments.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Exposures Without Assignments Chart
 description: This guide walks through troubleshooting exposures without assignments.
 ---
 
-The Exposures without Assignments chart appears in the **Monitor** tab and queries for the cumulative number of unique users who have performed an exposure event without a corresponding assignment event within each day. 
+The Exposures without Assignments chart appears in the **Diagnostics** card and queries for the cumulative number of unique users who have performed an exposure event without a corresponding assignment event within each day.
```
Comment (akhil-prakash) [inline docs/experiment/guides/troubleshooting/exposures-without-assignments.md:6]: we no longer have tabs in the new UI. and the name changed from Monitor to Diagnostics
See https://app.amplitude.com/experiment/amplitude/187520/config/151918/activity

Comment (akhil-prakash) [inline docs/experiment/guides/troubleshooting/exposures-without-assignments.md:6]: @markzegarelli Not everyone is on the new UI. Is it still okay to update the docs or should we wait till the new UI is rolled out to everyone?

Comment (markzegarelli) [reply docs/experiment/guides/troubleshooting/exposures-without-assignments.md:6]: @akhil-prakash Do you have a timeline for when we'll roll it out to everyone?

Comment (akhil-prakash) [reply docs/experiment/guides/troubleshooting/exposures-without-assignments.md:6]: Roughly first week of February

Comment (markzegarelli) [reply docs/experiment/guides/troubleshooting/exposures-without-assignments.md:6]: Cool. I think we should hold off until then, or until we meet some threshold of users (50%?) that have the new UI.

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**